pkg/render/render.go

```diff
@@ -28,6 +28,19 @@ import (
 	"go.uber.org/zap"
 )
 
+// allowedResponseCodes are the list of allowed response codes. This is
+// primarily here to catch if someone, in the future, accidentially includes a
+// bad status code.
+var allowedResponseCodes = map[int]struct{}{
```

Comment:
Do these need to be a map of structs? Would a set of ints not be able to represent the allowed codes?

Reply:
Go structs are 0 bytes, so a map[int]struct{} is essentially a slice with O(1) lookups